.github/workflows/main_housingrepairsschedulingapi.yml

```diff
@@ -27,6 +27,13 @@ jobs:
         run: dotnet tool update --global dotnet-format
       - name: Run dotnet format
         run: dotnet format --check --verbosity diagnostic
+      - name: Terraform fmt

```

Comment:
Could we name this Format Terraform or something similar, just so that it's clear what the job will do? `fmt` looks like an acronym of some kind. 😅

Reply:
Done in [239a068](https://github.com/Newarkandsherwood/HousingRepairsSchedulingApi/pull/7/commits/239a06895bd0ea127ba75dee8e21341532177528)